<launch>

<include file="$(find robot_control)/launch/shared/sas_operator_side_receiver.launch"/>
<include file="$(find robot_control)/launch/shared/sas_patient_side_manager.launch"/>

<node pkg="robot_control" type="semi_autonomous_tele_operation.py" name="tele_controller" output="screen"/>

<node pkg="sas_datalogger" type="sas_datalogger_node.py" name="sas_datalogger" output="screen"/>
<node pkg="robot_control" type="store_node_semi_autonomous.py" name="store" output="screen"/>

</launch>
